File: Src/API/API/Model/Team.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Model
{
    class Team
    {
        List<User> Users;
        string TeamName;
        public Team(string teamname, List<User> users)
        {
            this.TeamName = teamname;
            this.Users = users;
        }
    }
}

File: MobApp/app/src/main/java/be/ap/eaict/geocapture/Team.java
package be.ap.eaict.geocapture;

import java.util.List;

public class Team {
    public int Id ;
    public List<User> Users ;
    public List<Locatie> CapturedLocaties;
    public String TeamName ;

    public Team(List<User> Users, List<Locatie> capturedLocaties, String teamName)
    {
        this.Users = Users;
        this.CapturedLocaties = capturedLocaties;
        this.TeamName = teamName;
    }
}
